docs/configure-custom-values-scalar-admin-k8s.md

```diff
@@ -0,0 +1,101 @@
+# Configure a custom values file for Scalar Admin k8s
+
+This document explains how to create your custom values file for the Scalar Admin k8s chart. For details on the parameters, see the [README](https://github.com/scalar-labs/helm-charts/blob/main/charts/scalar-admin-k8s/README.md) of the Scalar Admin k8s chart.
+
+## Required configurations
+
+This section explains the required configurations when setting up a custom values file for Scalar Admin k8s.
+
+### Flag configurations
+
+You must specify several flags to `scalarAdminK8s.commandArgs` as an array to run Scalar Admin k8s. For more details on the flags, see [README](https://github.com/scalar-labs/scalar-admin-k8s/blob/main/README.md) of Scalar Admin k8s.
```

Comment:
If we change the file name, please remember to change it here as well.

```suggestion
You must specify several flags to `scalarAdminForKubernetes.commandArgs` as an array to run Scalar Admin for Kubernetes. For more details on the flags, see [README](https://github.com/scalar-labs/scalar-admin-k8s/blob/main/README.md) of Scalar Admin for Kubernetes.
```

Reply:
I updated the file path in the documents in 4c52c707aa514ed5ecef0edb5c21c61da3589417 .